### PR TITLE
Adding Travis CI

### PR DESCRIPTION
**Pull request by bvandersloot**
Can you merge this in and enable Travis CI for this repo? 

https://travis-ci.org/

This will give us status of if tests in `./tapdance/` pass or fail automatically.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+language: go
+
+go:
+ - 1.6
```
Comment (sergeyfrolov) [inline .travis.yml]: It's 1.7.4+

Comment (sergeyfrolov) [reply .travis.yml]: Is there a reason to use 1.6?

**Comment by sergeyfrolov** [comment]
Many thanks!

File: .travis.yml (hunk 1 above)
Comment (bvandersloot) [reply .travis.yml]: 1.6 is just what I am using. We can go with 1.7.4. `release` should cover the latest versions too.

**Comment by sergeyfrolov** [comment]
It looks like travis doesn't know what to build.

**Pull request merged**